File: uncategorized/1877_minimize_maximum_pair_sum_in_array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minPairSum = function(nums) {

    nums.sort((a,b) => a-b)
    let ans = 0

    for(let i=0; i<nums.length/2; i++){
        ans = Math.max(ans, nums[i] + nums[nums.length - (i+1)])
    }
    return ans
};

File: uncategorized/1877_minimize_maximum_pair_sum_in_array.go
func minPairSum(nums []int) int {

    sort.Ints(nums)
    ans := 0

    for i:=0; i<len(nums)/2; i++ {
        ans = max(ans, nums[i] + nums[len(nums)-(i+1)])
    }

    return ans

}